src/rest.js

```diff
@@ -151,8 +152,10 @@ function del(config, auth, className, objectId) {
         return;
       }
     })
-    .then(() => {
-      var options = {};
+    .then(() => config.database.loadSchema())
+    .then(s => {
+      schemaController = s;
```

Comment:
why?  why not just `((schemaController) => {`

Reply:
Need to save a reference to be used below.